tests/test_hub_mixin_pytorch.py

```diff
@@ -48,16 +50,25 @@ def setUpClass(cls):
     def test_save_pretrained_basic(self):
         DummyModel().save_pretrained(self.cache_dir)
         files = os.listdir(self.cache_dir)
-        self.assertTrue("pytorch_model.bin" in files)
+        self.assertTrue("model.safetensors" in files)
         self.assertEqual(len(files), 1)
 
     def test_save_pretrained_with_config(self):
         DummyModel().save_pretrained(self.cache_dir, config=CONFIG)
         files = os.listdir(self.cache_dir)
         self.assertTrue("config.json" in files)
-        self.assertTrue("pytorch_model.bin" in files)
+        self.assertTrue("model.safetensors" in files)
         self.assertEqual(len(files), 2)
 
+    def test_save_as_safetensors(self):
+        DummyModel().save_pretrained(self.cache_dir, config=TOKEN)
+        modelFile = self.cache_dir / "model.safetensors"
+        # check for safetensors header to ensure we are saving the model in safetensors format
+        # while an implementation detail, assert as this has safety implications
+        # https://github.com/huggingface/safetensors?tab=readme-ov-file#format
+        with open(modelFile, "rb") as f:
+            self.assertEqual(f.read(8), b"\x80\x00\x00\x00\x00\x00\x00\x00")
```

Comment:
```suggestion
            header_size = struct.unpack("<Q", f.read(8))[0]
        self.assertEqual(header_size, 128)
```

It's the same but more human-readable.